go.mod

```diff
@@ -16,6 +15,7 @@ require (
 require (
 	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6 // indirect
 	github.com/btcsuite/btcd v0.20.1-beta // indirect
+	github.com/centrifuge/go-substrate-rpc-client v2.0.0+incompatible // indirect
```

Comment:
Maybe this was automatically added somehow but this probably shouldn't exist as a dependency

Reply:
This `go mod tidy` command adds automatically as an indirect dependency of chainbridge-core. Nothing we can do about that.